src/main/java/com/synopsys/integration/alert/provider/blackduck/web/PolicyNotificationFilterCustomEndpoint.java

```diff
@@ -97,6 +96,8 @@ private boolean isFilterablePolicy(Collection<String> notificationTypes) {
     }
 
     private List<NotificationFilterModel> retrieveBlackDuckPolicyOptions() throws IntegrationException {
+        //FIXME have to figure out how to create the properties for this endpoint.
```

Comment:
Can we Autowire the BlackDuckPropertiesFactory?

Reply:
I realize we still need to have the config.

Reply:
I can autowire the factory.